### DISCUSSION TITLE
lr_config and num_input_frames set

### DISCUSSION THREAD

**Discussion by Winnie202** [open-mmlab/mmediting#1284] [How-to]
when i train restorers model，config is realbasicvsr_c64b20_1x30x8_lr5e-5_150k_reds.py，The loss is like this
![2022-07-13 10-34-17屏幕截图](https://user-images.githubusercontent.com/86768013/178644379-20755a1a-7005-4faf-8805-0970561959ea.png)
Please how to solve this problem

**Comment by Winnie202** [comment]
How should the value of lr_config and num_input_frames be set? Is the value of num_input_frames  the bigger the better?
![2022-07-12 11-04-48屏幕截图](https://user-images.githubusercontent.com/86768013/178403213-ff809d63-2e6e-44ff-832a-e59f462c94de.png)

**Comment by zengyh1900** [comment]
@Z-Fran 
Please check this issue.
